packages/connectivity/src/scp-cf/authorization-header.ts

```diff
@@ -147,6 +147,22 @@ function headerForProxy(
   }
 }
 
+function noAuthOnPremiseProxy(
+  destination: Destination
+): Record<string, any> {
+  let principalPropagationHeader;
+  try {
+    principalPropagationHeader = headerForPrincipalPropagation(destination);
+  } catch (e) {
+    logger.warn('No principal propagation header found.');
```

Comment:
Yeah, I think you could argue that this is rather a debug message, I believe it was a warning because previously NoAuth was used as some principal propagation workaround, so if those headers were missing a warning was thrown, I will convert it into a debug message.